lib/stimulus_reflex.rb

```diff
@@ -20,5 +20,53 @@
 
 module StimulusReflex
   class Engine < Rails::Engine
+    NODE_VERSION_FORMAT = /(\d\.\d\.\d.*):/
```

Comment:
Should this be `/(\d{1,2}\.\d{1,2})\.\d{1,2}.*):/`

Or if you ever think you're going to go above 2 digits you could use `\d+`

Currently, `\d` will only match the first number, but will not match if there is a 2nd digit.

Reply:
Good call, that should be `\d+` for each. 👍